requirements.txt

```diff
@@ -1,5 +1,5 @@
 # Based on the output from "pip freeze"
-requests==2.20.0
+requests==2.31.0
```

Comment:
```suggestion
# Blinly update from request 2.20.0 to avoid github security alert. Not sure if the code will work as is
# after the upgrade but probably this repository is no longer used and it is not a big issue ;)
requests==2.31.0
```